### PR TITLE
44 playwright

### PR DESCRIPTION
**Pull request by nictownsend**
## Status
**READY/IN DEVELOPMENT/HOLD**

### Commit Message Title
* When opening this PR, the raiser should set the title of this PR to the first line of their desired commit message, e.g:
```
feat|fix|docs|style|refactor|perf|test|chore: changed function X
```
* The reviewer should ensure that the first commit message field is of this form when performing the `squash and merge` from this page.

### Commit Message Description
```
# When opening this PR, the raiser should replace this text with the remaining
# lines of their desired commit message, e.g:

Further details of the code going into the commit

Contributes to: #XYZ
Closes: #XYZ

Signed-off-by: Your Name <email@address.com>
```
* The reviewer should copy the above text into the extended description field when performing the `squash and merge` from this page.

## Checklist
- [ ] Automated tests exist
- [ ] Documentation exists [link]()
- [ ] Local unit tests performed
- [ ] Sufficient logging/trace
- [ ] Desired commit message set as PR title and commit description set above

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -51,7 +52,7 @@
       "footer-max-length": [
         2,
         "always",
-        72
+        100
```
Comment (matthew-chirgwin) [inline ui/package.json:55]: What was this for?

File: ui/docs/Test.md [hunk 2]
```diff
@@ -53,9 +53,21 @@ React component code:
...
+End to End integration tests should exercise a user scenario - focus on the behaviour a user will take to interact with the application to achieve an end goal.
+
+They should be fairly coarse grained, exercising multiple pages/panels as necessary.
+
+For example - "I can produce a message and observe it being consumed", or "I can stop producing and no more messages are consumed".
+
+These tests are using _Jest_ as a test runner, using standard Jest test assets,
+such as `describe`, `it` and `expect`, and _Playwright_ to instrument the browser.
+
+To avoid the temptation to correlate scenarios with particular panels, these tests are located in the `e2e/test` directory.
```
Comment (matthew-chirgwin) [inline ui/docs/Test.md:67]: No issue with this - would you mind checking if there is any mention of e2e tests in any of the other .md files/the Bootsrtrap folder readme and updating as required?

File: ui/package.json (hunk 1 above)
Comment (nictownsend) [reply ui/package.json:55]: To allow the `Contributes-to:` to work (seems like the last two lines counted as the footer) - happy to remove however as I've not got any proof as to "why" it works

File: ui/docs/Test.md (hunk 2 above)
Comment (nictownsend) [reply ui/docs/Test.md:67]: Sure, on it :) Will address in the cucumber PR if that's ok?

File: ui/package.json (hunk 1 above)
Comment (matthew-chirgwin) [reply ui/package.json:55]: If it allows `Contributes-to:` I have no objections!

File: ui/docs/Test.md (hunk 2 above)
Comment (matthew-chirgwin) [reply ui/docs/Test.md:67]: Given there is bound to be more churn (and you are the only one on this atm) I dont mind!

**Pull request merged**